Client/Frontend/Shields/ShieldsViewController.swift

```diff
@@ -336,4 +361,30 @@ class ShieldsViewController: UIViewController, PopoverContentComponent, Themeabl
     required init?(coder aDecoder: NSCoder) {
         fatalError()
     }
+    
+    // MARK: Share Actions
+    
+    func shareTrackersAndAdsWithEmail() {
```

Comment:
we have similar methods in BVC already, do you think it would make sense to pass it to bvc via some kind of delegate?

Reply:
I know but I prefer them separate, they are probably going to have different share events in the end but according to upcoming requirement I can create a Provider for the share information.

Reply:
Yeah fair comment but I wanna keep in this way for now until we received what to share in certain scenarios, I might create a Provider for information will be shared.